src/contracts/misc/GhoAaveSteward.sol

```diff
@@ -0,0 +1,117 @@
+import {DataTypes} from '@aave/core-v3/contracts/protocol/libraries/types/DataTypes.sol';
+import {ReserveConfiguration} from '@aave/core-v3/contracts/protocol/libraries/configuration/ReserveConfiguration.sol';
+import {GhoInterestRateStrategy} from '../facilitators/aave/interestStrategy/GhoInterestRateStrategy.sol';
+import {IFixedRateStrategyFactory} from '../facilitators/aave/interestStrategy/interfaces/IFixedRateStrategyFactory.sol';
+import {IGhoAaveSteward} from './interfaces/IGhoAaveSteward.sol';
+import {RiskCouncilControlled} from './RiskCouncilControlled.sol';
+
+/**
+ * @title GhoAaveSteward
+ * @author Aave Labs
+ * @notice Helper contract for managing parameters of the GHO reserve
+ * @dev Only the Risk Council is able to action contract's functions, based on specific conditions that have been agreed upon with the community.
+ * @dev Requires role RiskAdmin on the Aave V3 Ethereum Pool
+ */
+contract GhoAaveSteward is Ownable, RiskCouncilControlled, IGhoAaveSteward {
+  using ReserveConfiguration for DataTypes.ReserveConfigurationMap;
+
+  /// @inheritdoc IGhoAaveSteward
+  uint256 public constant MINIMUM_DELAY = 2 days;
+
+  /// @inheritdoc IGhoAaveSteward
+  address public immutable POOL_ADDRESSES_PROVIDER;
+
+  /// @inheritdoc IGhoAaveSteward
+  address public immutable GHO_TOKEN;
+
+  /// @inheritdoc IGhoAaveSteward
+  address public immutable FIXED_RATE_STRATEGY_FACTORY;
+
+  /// @inheritdoc IGhoAaveSteward
+  function RISK_COUNCIL() public view override returns (address) {
+    return COUNCIL;
+  }
+
+  GhoDebounce internal _ghoTimelocks;
+
+  /**
+   * @dev Only methods that are not timelocked can be called if marked by this modifier.
+   */
+  modifier notTimelocked(uint40 timelock) {
+    require(block.timestamp - timelock > MINIMUM_DELAY, 'DEBOUNCE_NOT_RESPECTED');
+    _;
+  }
+
+  /**
+   * @dev Constructor
+   * @param owner The address of the owner of the contract
+   * @param addressesProvider The address of the PoolAddressesProvider of Aave V3 Ethereum Pool
+   * @param ghoToken The address of the GhoToken
+   * @param fixedRateStrategyFactory The address of the FixedRateStrategyFactory
+   * @param riskCouncil The address of the risk council
+   */
+  constructor(
+    address owner,
+    address addressesProvider,
+    address ghoToken,
+    address fixedRateStrategyFactory,
+    address riskCouncil
+  ) RiskCouncilControlled(riskCouncil) {
+    require(owner != address(0), 'INVALID_OWNER');
+    require(addressesProvider != address(0), 'INVALID_ADDRESSES_PROVIDER');
+    require(ghoToken != address(0), 'INVALID_GHO_TOKEN');
+    require(fixedRateStrategyFactory != address(0), 'INVALID_FIXED_RATE_STRATEGY_FACTORY');
+
+    POOL_ADDRESSES_PROVIDER = addressesProvider;
+    GHO_TOKEN = ghoToken;
+    FIXED_RATE_STRATEGY_FACTORY = fixedRateStrategyFactory;
+
+    _transferOwnership(owner);
+  }
+
+  /// @inheritdoc IGhoAaveSteward
+  function updateGhoBorrowCap(
```

Comment:
Let's add `updateGhoSupplyCap` as well. This reverts if the reserve mints GHO (like the one on Ethereum), so let's create a test for that.

Reply:
Added. Unsure about the test case you mention. Let me know if it's covered now

Reply:
GHO can work on Aave as a Facilitator (no supply, GHO is minted upon borrow) or as a regular reserve (LPs can supply/withdraw, borrow take GHO out from the liquidity pool).
Setting a supply cap only has effect on the latter case. We must add these test cases